### PR TITLE
Flight results integration

### PR DESCRIPTION
**Pull request by Chris-DJ-Kim**
Flight results for both incoming and outbound flights are obtained from flight scanner api. Details for both routes (if available) are returned. Only rudimentary styling on returned details.

### CODE REVIEW

File: client/src/components/Search.js [hunk 1]
```diff
@@ -57,16 +57,16 @@ const Search = (props) => {
     setState({ ...state, [e.target.name]: e.target.value });
   };
   const handleSubmit = async () => {
```
Comment (vi-nguyen-12) [inline client/src/components/Search.js:59]: I just pushed an axios file for FE to call Api easier. would you have a look and apply it here. it's like:
 const handleSubmit=async ()={
const response= await skyscannerApi.getQuotes(data);
...
}
It's just my opinion, what do you think?

Comment (Chris-DJ-Kim) [reply client/src/components/Search.js:59]: I took a look and looks good! I'll need to wait until your files are merged as there are likely edits I need to make to the front end components to accommodate. Once you merge I will update from the dev and dig into it!

File: client/src/components/flight-details-by-direction.component.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+      borderBottom: "1px solid #D3D3D3",
+    },
+    icon: {
+      fontSize: "40px",
+      paddingRight: "15px",
+    },
+    button: {
+      backgroundColor: "#ffa500",
+      "&:hover": {
+        backgroundColor: "#ff8c00",
```
Comment (jusa798) [inline client/src/components/flight-details-by-direction.component.js]: I like that you thought about the hover. You could also probably add the colors to the theme so we can keep the styling consistent. Nice work on the timeline!

File: client/src/components/flight-details-by-direction.component.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+    icon: {
+      fontSize: "40px",
+      paddingRight: "15px",
+    },
+    button: theme.buttonPrimary,
+  });
+  const classes = useStyles();
+
+  const { details, quoteDetails, extraInfo } = props;
+  return (
```
Comment (sina-jamshidi) [inline client/src/components/flight-details-by-direction.component.js:27]: nice, easy to read

File: client/src/components/flight-details.component.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+      textAlign: "center",
+      marginTop: "35px",
+      border: "1px solid #D3D3D3",
+      borderRadius: "5px",
+    },
+  });
+  const classes = useStyles();
+
+  const { quoteDetails, places, carriers } = props;
+  //THIS DEFINTELY NEEDS A UTIL FUNCTION OF ITS OWN LATER
```
Comment (sina-jamshidi) [inline client/src/components/flight-details.component.js]: Can't we combine this into getFlightDetails and just have it return two objects: `outgoingDetails` and `inboundDetails`?

File: client/src/utils/skyscanner.js [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+    headers: {
+      "x-rapidapi-key": "12ace0bfd9msh34dee4306bb93d3p1da922jsn4c10b1bd8ee4",
+      "x-rapidapi-host":
+        "skyscanner-skyscanner-flight-search-v1.p.rapidapi.com",
+      useQueryString: true,
+    },
+  });
+  //Places[0] is the city. Places[1] is the first, usually major airport
+  return response.data.Places[1].PlaceId;
+};
```
Comment (sina-jamshidi) [inline client/src/utils/skyscanner.js:20]: we can leave this for now since there will be autocomplete as a separate task later

**Review by sina-jamshidi — APPROVED** [review]
Looks good!
Chloe's PR will be merged soon, so you can update this.

File: client/src/components/flight-details.component.js (hunk 4 above)
Comment (Chris-DJ-Kim) [reply client/src/components/flight-details.component.js]: Yes, just combined it! The repetitive part is now handled within the getFlightDetails function.

Comment (Chris-DJ-Kim) [reply client/src/components/flight-details.component.js]: Actually, getFlightDetails was repetitive to begin with so not much changed on that part

**Comment by Chris-DJ-Kim** [comment]
If anyone wonders why there are two reverts in the commit history...there was a misclick.

**Pull request merged**